itou/metabase/management/commands/populate_metabase_emplois.py

```diff
@@ -238,71 +242,93 @@ def populate_job_descriptions(self):
             .all()
         )
 
-        self.populate_table(table=job_descriptions.TABLE, querysets=[queryset])
+        self.populate_table(job_descriptions.TABLE, batch_size, querysets=[queryset])
 
-    def populate_organizations(self):
+    def populate_organizations(self, batch_size):
         """
         Populate prescriber organizations,
         and add a special "ORG_OF_PRESCRIBERS_WITHOUT_ORG" to gather stats
         of prescriber users *without* any organization.
         """
-        queryset = PrescriberOrganization.objects.prefetch_related(
-            "prescribermembership_set", "members", "jobapplication_set"
+        active_user_created_job_applications_filter = Q(
+            jobapplication__created_from_pe_approval=False,
+            jobapplication__to_siae_id__in=get_active_siae_pks(),
+        )
+        job_applications_count = Count(
+            "jobapplication",
+            filter=active_user_created_job_applications_filter,
```

Comment:
distinct=True obligatoire

Reply:
_And the Titanic did not hit the iceberg..._